### PR TITLE
Rh working

### PR DESCRIPTION
**Pull request by ruoyinghao**
*renamed executable main instead of game
*included Russel's code, created .cpp counterparts
*moved code away from main (but they stuck in game.cpp because I encountered issue when letting MenuScreen accepting window pointer)
*DEFINITION.hpp is now included in global.hpp
*bring back the asset (background, font) that did not committed into repo
*now we have a decent looking start menu

### CODE REVIEW

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (addtheletters) [inline .idea/codeStyleSettings.xml:1]: Could we add .idea to .gitignore, or is having this in the repo helpful?

File: include/global.hpp [hunk 2]
```diff
@@ -8,5 +8,10 @@
 #include <iostream>
 #include <cmath>
```
Comment (addtheletters) [inline include/global.hpp:9]: Can we put ifndef/define guards? I don't think pragma once is standard

File: src/Game.cpp [hunk 3]
```diff
@@ -0,0 +1,85 @@
+//
+// Created by Ruoying Hao on 10/17/17.
+//
+
+#include "../include/Game.hpp"
```
Comment (addtheletters) [inline src/Game.cpp:5]: I think it should be okay to just say Game.hpp?

File: .idea/codeStyleSettings.xml (hunk 1 above)
Comment (ruoyinghao) [reply .idea/codeStyleSettings.xml:1]: that's a good idea

File: src/Game.cpp (hunk 3 above)
Comment (ruoyinghao) [reply src/Game.cpp:5]: True

**Pull request merged**